### ISSUE TITLE
perlfunc chmod entry

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#7650]
Migrated from [rt.perl.org#32800](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=32800) (status was 'resolved')

Searchable as RT32800$

**Comment by p5pRT** [comment]
### From brucer@cisco.com

#### Created by brucer@iceage.cisco.com


missing verb
'should \*not a string'
'should \*not be a string'

<details><summary>Perl Info</summary>

```

Flags:
    category=docs
    severity=low

Site configuration information for perl v5.8.5:

Configured by brucer at Fri Sep 17 11:38:36 PDT 2004.

Summary of my perl5 (revision 5 version 8 subversion 5) configuration:
  Platform:
    osname=solaris, osvers=2.8, archname=sun4-solaris
    uname='sunos iceage 5.8 generic_108528-17 sun4u sparc sunw,sun-fire-880 '
    config_args='-Dcc=gcc -Dprefix=/users/brucer/sparc-sun-solaris2.8 -des'
    hint=recommended, useposix=true, d_sigaction=define
    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
    useperlio=define d_sfio=undef uselargefiles=define usesocks=undef
    use64bitint=undef use64bitall=undef uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='gcc', ccflags ='-fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64',
    optimize='-O',
    cppflags='-fno-strict-aliasing -pipe -I/usr/local/include'
    ccversion='', gccversion='3.4.2', gccosandvers='solaris2.8'
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=4321
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=8, prototype=define
  Linker and Libraries:
    ld='gcc', ldflags =' -L/usr/local/lib '
    libpth=/usr/local/lib /usr/lib /usr/ccs/lib
    libs=-lsocket -lnsl -lgdbm -ldb -ldl -lm -lc
    perllibs=-lsocket -lnsl -ldl -lm -lc
    libc=/lib/libc.so, so=so, useshrplib=false, libperl=libperl.a
    gnulibc_version=''
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags=' '
    cccdlflags='-fPIC', lddlflags='-G -L/usr/local/lib'

Locally applied patches:
    


@INC for perl v5.8.5:
    /users/brucer/sparc-sun-solaris2.8/lib/perl5/5.8.5/sun4-solaris
    /users/brucer/sparc-sun-solaris2.8/lib/perl5/5.8.5
    /users/brucer/sparc-sun-solaris2.8/lib/perl5/site_perl/5.8.5/sun4-solaris
    /users/brucer/sparc-sun-solaris2.8/lib/perl5/site_perl/5.8.5
    /users/brucer/sparc-sun-solaris2.8/lib/perl5/site_perl
    .


Environment for perl v5.8.5:
    HOME=/users/brucer
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH=/usr/local/lib:
    LOGDIR (unset)
    PATH=.:/users/brucer/sparc-sun-solaris2.8/bin:/usr/java/j2re1.4.1_01/bin:/users/brucer/share/bin:/auto/gsg-sw/inst/share/bin:/router/bin:/usr/local/packages/atria/current/contrib:/sw/current/solaris2bin:/usr/local/bin:/usr/atria/bin:/nfs/ddts/ddtshome/bin:/bin:/usr/bin:/auto/gsg-sw/inst/share/bin:/auto/gsg-sw/share/bin:/auto/gsg-sw/inst/solaris/bin:/auto/gsg-sw/share:/auto/gsg-sw/linuxbin:/auto/gsg-tools/frame/current/bin:/opt/ECSossh/bin:/usr/bin:/usr/sbin:/usr/openwin/bin:/usr/local/bin
    PERL_BADLANG (unset)
    SHELL=/bin/bash

```
</details>

**Comment by p5pRT** [comment]
### From @smpeters

> \[brucer@&#8203;cisco\.com \- Fri Dec 03 12&#8203;:40&#8203;:05 2004\]&#8203;:
> 
> 
> This is a bug report for perl from brucer@&#8203;iceage\.cisco\.com\,
> generated with the help of perlbug 1\.35 running under perl v5\.8\.5\.
> 
> 
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> \[Please enter your report here\]
> missing verb
> 'should \*not a string'
> 'should \*not be a string'
> 
> \[Please do not change anything below this line\]
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> \-\-\-
> Flags&#8203;:
>     category=docs
>     severity=low
> \-\-\-
> Site configuration information for perl v5\.8\.5&#8203;:
> 
> Configured by brucer at Fri Sep 17 11&#8203;:38&#8203;:36 PDT 2004\.
> 
> Summary of my perl5 \(revision 5 version 8 subversion 5\) configuration&#8203;:
>   Platform&#8203;:
>     osname=solaris\, osvers=2\.8\, archname=sun4\-solaris
>     uname='sunos iceage 5\.8 generic\_108528\-17 sun4u sparc sunw\,sun\-
>    fire\-880 '
>     config\_args='\-Dcc=gcc \-Dprefix=/users/brucer/sparc\-sun\-solaris2\.8
>    \-des'
>     hint=recommended\, useposix=true\, d\_sigaction=define
>     usethreads=undef use5005threads=undef useithreads=undef
>    usemultiplicity=undef
>     useperlio=define d\_sfio=undef uselargefiles=define usesocks=undef
>     use64bitint=undef use64bitall=undef uselongdouble=undef
>     usemymalloc=n\, bincompat5005=undef
>   Compiler&#8203;:
>     cc='gcc'\, ccflags ='\-fno\-strict\-aliasing \-pipe
>    \-I/usr/local/include \-D\_LARGEFILE\_SOURCE \-D\_FILE\_OFFSET\_BITS=64'\,
>     optimize='\-O'\,
>     cppflags='\-fno\-strict\-aliasing \-pipe \-I/usr/local/include'
>     ccversion=''\, gccversion='3\.4\.2'\, gccosandvers='solaris2\.8'
>     intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=4321
>     d\_longlong=define\, longlongsize=8\, d\_longdbl=define\,
>    longdblsize=16
>     ivtype='long'\, ivsize=4\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\,
>    lseeksize=8
>     alignbytes=8\, prototype=define
>   Linker and Libraries&#8203;:
>     ld='gcc'\, ldflags =' \-L/usr/local/lib '
>     libpth=/usr/local/lib /usr/lib /usr/ccs/lib
>     libs=\-lsocket \-lnsl \-lgdbm \-ldb \-ldl \-lm \-lc
>     perllibs=\-lsocket \-lnsl \-ldl \-lm \-lc
>     libc=/lib/libc\.so\, so=so\, useshrplib=false\, libperl=libperl\.a
>     gnulibc\_version=''
>   Dynamic Linking&#8203;:
>     dlsrc=dl\_dlopen\.xs\, dlext=so\, d\_dlsymun=undef\, ccdlflags=' '
>     cccdlflags='\-fPIC'\, lddlflags='\-G \-L/usr/local/lib'
> 
> Locally applied patches&#8203;:
> 
> 
> \-\-\-
> @&#8203;INC for perl v5\.8\.5&#8203;:
>     /users/brucer/sparc\-sun\-solaris2\.8/lib/perl5/5\.8\.5/sun4\-solaris
>     /users/brucer/sparc\-sun\-solaris2\.8/lib/perl5/5\.8\.5
>     /users/brucer/sparc\-sun\-solaris2\.8/lib/perl5/site\_perl/5\.8\.5/sun4\-
>    solaris
>     /users/brucer/sparc\-sun\-solaris2\.8/lib/perl5/site\_perl/5\.8\.5
>     /users/brucer/sparc\-sun\-solaris2\.8/lib/perl5/site\_perl
>     \.
> 
> \-\-\-
> Environment for perl v5\.8\.5&#8203;:
>     HOME=/users/brucer
>     LANG \(unset\)
>     LANGUAGE \(unset\)
>     LD\_LIBRARY\_PATH=/usr/local/lib&#8203;:
>     LOGDIR \(unset\)
>     PATH=\.&#8203;:/users/brucer/sparc\-sun\-
>   
solaris2\.8/bin&#8203;:/usr/java/j2re1\.4\.1\_01/bin&#8203;:/users/brucer/share/bin&#8203;:/auto/gsg\-
>   
sw/inst/share/bin&#8203;:/router/bin&#8203;:/usr/local/packages/atria/current/contrib&#8203;:/sw/current/solaris2bin&#8203;:/usr/local/bin&#8203;:/usr/atria/bin&#8203;:/nfs/ddts/ddtshome/bin&#8203;:/bin&#8203;:/usr/bin&#8203;:/auto/gsg\-
>    sw/inst/share/bin&#8203;:/auto/gsg\-sw/share/bin&#8203;:/auto/gsg\-
>    sw/inst/solaris/bin&#8203;:/auto/gsg\-sw/share&#8203;:/auto/gsg\-
>    sw/linuxbin&#8203;:/auto/gsg\-
>   
tools/frame/current/bin&#8203;:/opt/ECSossh/bin&#8203;:/usr/bin&#8203;:/usr/sbin&#8203;:/usr/openwin/bin&#8203;:/usr/local/bin
>     PERL\_BADLANG \(unset\)
>     SHELL=/bin/bash
> 
> 

The follow patch fixes this gramatical error\.

<details><summary>Inline Patch</summary>

```diff
--- perlfunc.pod.orig   Fri Dec  3 22:03:44 2004
+++ perlfunc.pod        Fri Dec  3 22:05:49 2004
@@ -614,7 +614,7 @@
 
 Changes the permissions of a list of files.  The first element of the
 list must be the numerical mode, which should probably be an octal
-number, and which definitely should I<not> a string of octal digits:
+number, and which definitely should I<not> be a string of octal digits:
 C<0644> is okay, C<'0644'> is not.  Returns the number of files
 successfully changed.  See also L</oct>, if all you have is a string.
```
</details>

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @rgs

Steve Peters via RT wrote&#8203;:
> The follow patch fixes this gramatical error\.
> 
> \-\-\- perlfunc\.pod\.orig   Fri Dec  3 22&#8203;:03&#8203;:44 2004
> \+\+\+ perlfunc\.pod        Fri Dec  3 22&#8203;:05&#8203;:49 2004

Thanks\, fixed \(\#23637\)\.

**Comment by p5pRT** [comment]
@rgs - Status changed from 'open' to 'resolved'
